CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ Fixed - for any bug fixes.
 Security - in case of vulnerabilities.
 -->
 
+## [2.9.0] 2020-08-03
```

Comment:
It should be in `Unreleased`

Reply:
Or if you gonna release than you also need to update `Client` constant

Reply:
thanks. got it 👍